Themes/Views/Account/Login.cshtml

```diff
@@ -20,11 +20,14 @@
                     <input class="govuk-input govuk-input--width-20" asp-for="EmailAddress" spellcheck="false" autocomplete="email">
                 </div>
                 <div class="govuk-form-group govuk-!-margin-bottom-2" asp-validation-error-class-for="Password">
-                    <label class="govuk-label" asp-for="Password">
+                    <label class="govuk-label" for="password">
```

Comment:
@sunilbeis, can you double check this? I know you've just added asp-for to a lot of label elements.

Reply:
asp-for is correct

Reply:
change label back to asp-for="Password"